index.html

```diff
@@ -30,6 +30,148 @@ <h6><a href="#">LET’S CONNECT</a></h6>
                 </ul>
             </div>
```

Comment:
- [ ] Your social media icons from lines 25 to 29 are not displaying correctly, kindly add a `.` at the beginning of every link `./images/Vector.svg" alt="twitter`
- [ ] Please replace `My logo` with your own name or logo to personalize your portfolio.
- [ ] Kindly, for the main container of the project use display:grid. You can refer to [this](https://www.w3schools.com/css/css_grid_item.asp) article for more informations.

- [ ] Please add a background image so that your design will match the Figma design